doc/doc_homepage.md

```diff
@@ -35,6 +35,7 @@ You have different options:
 Star Spoon on Github: [https://github.com/INRIA/spoon](https://github.com/INRIA/spoon). :-)
 
 <!-- .* Marker comment. -->
+- November 17, 2017: Spoon 6.0.0 is released! Check the [(changelog)](https://github.com/INRIA/spoon/releases/tag/spoon-core-6.0.0) as this major release may introduce breaking changes :warning:
```

Comment:
I would say "as this major release may introduce breaking changes :warning:" here because we actually break very little

I'd say "there are few non backward-compatible changes, see changes"